packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -381,6 +381,16 @@ describe('rum public api', () => {
       expect(errorSpy).not.toHaveBeenCalled()
     })
 
+    it('should remove the user', () => {
+      rumPublicApi.removeUser()
```

Comment:
We should probably add a user first, otherwise this test would pass even if we don't call `removeUser`